testing/generate-custom-metadata/src/main.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2019-2023 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+use codec::Encode;
+
+/// can be called from the root of the project with: `cargo run --bin generate-custom-metadata`.
+/// Generates a "./artifacts/metadata_with_custom_values.scale" file.
+fn main() {
+    let metadata_prefixed = generate_custom_metadata::metadata_custom_values_foo();
+    let bytes = metadata_prefixed.encode();
+    std::fs::write("./artifacts/metadata_with_custom_values.scale", bytes)
```

Comment:
Can we add a call to this to `/scripts/artifacts.sh` (ie `cargo run --bin generate-custom-metadata > artifacts/metadata_with_custom_values.scale`), so that running that script updates all of our artifacts :)

(also, let's either allow the caller to decide whether to write the file to, or write the file to stdout, so that the file location can be specified in `artifacts.sh` so it's clear from that script what is being generated)

Reply:
Oh yes, that is a great idea!